### PR TITLE
Moved anchor links to the left on the pipeline parameter page 

### PR DESCRIPTION
**Pull request by sofiahag**
#1648

I just moved the anchor links to the left, to the beginning of the row. 

I tried making the fa symbol the anchor link as discussed on [slack](https://nfcore.slack.com/archives/C01QPMKBYNR/p1678782239632079?thread_ts=1678782186.643409&channel=C01QPMKBYNR&message_ts=1678782239.632079), but then it's not obvious that the fa symbol is the anchor link 🤔 

I'm open to input though, I don't feel strongly about it.

I'm aware that the anchor link pushed the parameter a bit to the right, but I think it looks OK still, or do you think I should push it back to the left?

<a href="https://gitpod.io/#https://github.com/nf-core/nf-co.re/pull/1713"></a>

### CODE REVIEW

**Comment by jfy133** [comment]
image

Screenshot posted on slack by @sofiahag for reference!

**Review by jfy133 — APPROVED** [review]
LGTM from a visualisation perspective. I leave the PHP code to @mashehu

File: public_html/assets/scss/_nf-core.scss [hunk 1]
```diff
@@ -528,6 +528,9 @@ footer a:active {
   -moz-transition: opacity 0.25s ease-in-out;
   -webkit-transition: opacity 0.25s ease-in-out;
 }
+.parameter-link {
+  margin: 0 10px 0 0;
```
Comment (mashehu) [inline public_html/assets/scss/_nf-core.scss:532]:
```suggestion
```

File: includes/functions.php [hunk 2]
```diff
@@ -447,19 +447,26 @@ function ($matches) use ($heading_ids, $is_hidden) {
...
-                '<a href="#' .
-                $hid .
-                '" class="header-link scroll_to_link"><span class="fas fa-link fa-xs ms-1"></span></a></h' .
-                $matches[3] .
+            $to_return = '<h' . $matches[1] . ' id="' . $hid . '" class="' . $hidden_class . '">';
+            if(strpos($matches[2], '</i>')){
+                $to_return .= $to_return . 
+                    '<a href="#' .
+                    $hid .
+                    '" class="header-link parameter-link scroll_to_link"><span class="fas fa-link fa-xs ms-1"></span></a>' .
```
Comment (mashehu) [inline includes/functions.php:455]:
```suggestion
                    '" class="header-link parameter-link scroll_to_link"><span class="fas fa-link fa-xs me-1"></span></a>' .
```
Is that enough margin, or do we need more (can't currently test it 😬)

File: public_html/assets/scss/_nf-core.scss (hunk 1 above)
Comment (mashehu) [reply public_html/assets/scss/_nf-core.scss:532]: better to let bootstrap to this (see above)

**Review by mashehu — APPROVED** [review]

**Review by mashehu — APPROVED** [review]

File: includes/functions.php (hunk 2 above)
Comment (mashehu) [reply includes/functions.php]: how about always showing them? somehow I have the filling it will have weird gaps otherwise?

**Comment by mashehu** [comment]
@nf-core-bot fix linting

File: includes/functions.php (hunk 2 above)
Comment (sofiahag) [reply includes/functions.php]: You mean like this?

Skärmavbild 2023-03-29 kl  13 11 28

**Comment by sofiahag** [comment]
The sub headers also inherited the formatting, I think it should be fine now:


Skärmavbild 2023-03-29 kl  15 27 55

**Comment by sofiahag** [comment]
@nf-core-bot fix linting

**Pull request merged**